adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralRewardedAdRenderer.m

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterMintegralRewardedAdRenderer.h"
+#import "GADMAdapterMintegralUtils.h"
+#import "GADMediationAdapterMintegralConstants.h"
+#import "GADMAdapterMintegralExtras.h"
+#include <stdatomic.h>
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKReward/MTGRewardAd.h>
+#import <MTGSDKReward/MTGBidRewardAdManager.h>
+
+@interface GADMAdapterMintegralRewardedAdRenderer ()<GADMediationRewardedAd,MTGRewardAdLoadDelegate,MTGRewardAdShowDelegate>
+
+@end
+
+@implementation GADMAdapterMintegralRewardedAdRenderer {
+    /// The completion handler to call when the ad loading succeeds or fails.
+    GADMediationRewardedLoadCompletionHandler _adLoadCompletionHandler;
+    
+    /// Data used to render an rewarded ad.
```

Comment:
Can you update this to: `  /// Ad configuration for the ad to be loaded.`

The same applies to all formats.